docs/v2-docs-guide.md

```diff
@@ -6,12 +6,125 @@ sidebar_label: V2 Docs Guide
 
 <br />
 
-## Cards 
+
+## Excerpt
```

Comment:
Added an `Excerpt` and `MultiStepSnippet` above the wave to see what it looks like with a normal background.